docs/generate_units.py

```diff
@@ -0,0 +1,51 @@
+from crnn.TFNetworkRecLayer import RecLayer
+from crnn.TFNativeOp import RecSeqCellOp
```

Comment:
Better use the new-style import here: `from returnn. TFNativeOp import ...`.

Reply:
ok, then i will change the alias and the other script accordingly